preprocess_image.py

```diff
@@ -119,8 +120,9 @@ def __call__(self, image):
     parser.add_argument('path', type=str, help="path to image (png, jpeg, etc.)")
     parser.add_argument('--size', default=256, type=int, help="output resolution")
     parser.add_argument('--border_ratio', default=0.2, type=float, help="output border ratio")
+    parser.add_argument('--dont_recenter_resize', action='store_true', help="dont recenter and resize, helpful for objaverse")
```

Comment:
Double-negatives get confusing, so I recommend you instead create `recenter` and `resize` arguments of type `bool` that default to True...

Reply:
Done!